src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1277,6 +1277,25 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         return viewPort.Height();
     }
 
+    // Method Description:
+    // - Sets a incremented or decremented fontsize as value to the current tab.
+    void TermControl::KeyboardZoomHandler(int zoom)

```

Comment:
Can't this just be consolidated/share with `MouseZoomHandler` and become a generic `ZoomHandler` or something?

Reply:
You are right, its nearly a complete copy from the mouse handler. 
Would you mind if I would use the mouse handler for this operation?

Reply:
I don't mind a re-use. Same action, same handler no matter what the source of the event... at least in my mind.

Reply:
Ok, I'll change the implementation for this.

Reply:
Sure, go for it. He's more of the UI-code guru than me. I would bias toward his suggestions.